src/main/java/org/jboss/tools/intellij/openshift/utils/odo/OdoCli.java

```diff
@@ -352,7 +231,7 @@ public void createComponentBinary(String project, String application, String con
    * ensure that $HOME/.odo/config.yaml file exists so thar we can use service related commands.
    */
   private void ensureDefaultOdoConfigFileExists() {
-    File dir = new File(HOME_FOLDER, PLUGIN_FOLDER);
+    File dir = new File(HOME_FOLDER, Constants.PLUGIN_FOLDER);
```

Comment:
for readability, either use Constants.HOME_FOLDER or static import PLUGIN_FOLDER.